server/bundler/loader.js

```diff
@@ -114,14 +114,10 @@ function splitTemplate( path, section ) {
 }
 
 function getPathRegex( pathString ) {
-	var regex;
-
 	if ( pathString === '/' ) {
 		return JSON.stringify( pathString );
 	}
-
-	regex = utils.pathToRegExp( pathString );
-
+	const regex = utils.pathToRegExp( pathString );
 	return '/' + regex.toString().slice( 1, -1 ) + '/';
```

Comment:
See my above [comment](https://github.com/Automattic/wp-calypso/pull/11705#discussion_r104003009). tl/dr IMO it's all BS we put up with only to route to both the actually specified route, and that route with the `/*` wildcard attached to it 😄